lastpass/fetcher.py

```diff
@@ -20,6 +20,7 @@
 
 http = requests
 
+headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36'}
```

Comment:
I will merge this.
Then change the User-Agent to another appropriate value. For example,`lastpass-python/0.3.1`.

Reply:
Fixed: https://github.com/konomae/lastpass-python/pull/42